core/tests/protractor_desktop/additionalEditorFeatures.js

```diff
@@ -383,6 +385,390 @@ describe('Full exploration editor', function() {
+      });
+      await action.waitForAutosave();
+      await explorationEditorMainTab.setContent(async function(richTextEditor) {
+        await richTextEditor.appendPlainText('Content 37');
+      });
+      await action.waitForAutosave();
+      await explorationEditorMainTab.setContent(async function(richTextEditor) {
+        await richTextEditor.appendPlainText('Content 38');
+      });
+      await action.waitForAutosave();
+      await explorationEditorMainTab.setContent(async function(richTextEditor) {
+        await richTextEditor.appendPlainText('Content 39');
+      });
+      await action.waitForAutosave();
+      await explorationEditorMainTab.setContent(async function(richTextEditor) {
+        await richTextEditor.appendPlainText('Content 40');
+      });
+      await action.waitForAutosave();
+      await explorationEditorMainTab.setContent(async function(richTextEditor) {
+        await richTextEditor.appendPlainText('Content 41');
+      });
+      await action.waitForAutosave();
+      await explorationEditorMainTab.setContent(async function(richTextEditor) {
+        await richTextEditor.appendPlainText('Content 42');
+      });
+      await action.waitForAutosave();
+      await explorationEditorMainTab.setContent(async function(richTextEditor) {
+        await richTextEditor.appendPlainText('Content 43');
+      });
+      await action.waitForAutosave();
+      await explorationEditorMainTab.setContent(async function(richTextEditor) {
+        await richTextEditor.appendPlainText('Content 44');
+      });
+      await action.waitForAutosave();
+      await explorationEditorMainTab.setContent(async function(richTextEditor) {
+        await richTextEditor.appendPlainText('Content 45');
+      });
+      await action.waitForAutosave();
+      await explorationEditorMainTab.setContent(async function(richTextEditor) {
+        await richTextEditor.appendPlainText('Content 46');
+      });
+      await action.waitForAutosave();
+      await explorationEditorMainTab.setContent(async function(richTextEditor) {
+        await richTextEditor.appendPlainText('Content 47');
+      });
+      await action.waitForAutosave();
+      await explorationEditorMainTab.setContent(async function(richTextEditor) {
+        await richTextEditor.appendPlainText('Content 48');
+      });
+      await action.waitForAutosave();
+      await explorationEditorMainTab.setContent(async function(richTextEditor) {
+        await richTextEditor.appendPlainText('Content 49');
+      });
+      await action.waitForAutosave();
+      await explorationEditorMainTab.setContent(async function(richTextEditor) {
+        await richTextEditor.appendPlainText('Content 50');
+      });
+      await action.waitForAutosave();
+      await waitFor.visibilityOf(
+        element(by.css('.protractor-test-save-prompt-modal')),
+        'Save Prompt Modal taking too long to appear');
+      await explorationEditorPage.saveChangesAfterPrompt(
```

Comment:
Unclear what prompt is referred to here.

Reply:
Explained it now.